Cod3rsGrowth.Forms/Forms/FormCriarIngrediente.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Cod3rsGrowth.Dominio.Entidades;
+using Cod3rsGrowth.Dominio.Enums;
+using Cod3rsGrowth.Servico.Servicos;
+
+namespace Cod3rsGrowth.Forms
+{
+    public partial class FormCriarIngrediente : Form
+    {
+        private ServicoIngrediente _servicoIngrediente;
+        public FormCriarIngrediente(ServicoIngrediente servicoIngrediente)
+        {
+            InitializeComponent();
+            _servicoIngrediente = servicoIngrediente;
+            comboBox_Naturalidade.DataSource = Enum.GetValues(typeof(Naturalidade));
```

Comment:
Coloco isso em um método e chama ele dentro do evento load do seu form.

Reply:
No caso só a combo_box né, o serviço precisa tar no construtor

Reply:
Isso.

Reply:
Tudo que for recurso para seu formulário deve ser carregado a partir do evento load